Add antimicrobial resistance annotations

njohner opened the pull request:
With this PR we add a pipeline to extract AMR annotations using the [AMRFinderPlus](https://www.ncbi.nlm.nih.gov/pathogens/antimicrobial-resistance/AMRFinder/) software and add them to the database. I also added the annotations to the locus and orthogroup views.

Also note that:
- currently only docker is supported, will add conda support later.
- I had to fix a few f-strings that got broken by autopep8. It seems that pycodestyle does not correctly support f-strings.

Screenshot of the annotations on a locus tag view
![AMR_locus_view](https://github.com/metagenlab/zDB/assets/7374243/48e6cf22-3bf6-465f-a51d-97354ae83862)

tpillone commented:
Nice work! 
I would remove the scope and add the gene symbol to the table (with a link to the NCBI entry of the HMM, e.g https://www.ncbi.nlm.nih.gov/genome/annotation_prok/evidence/NF033374/)

njohner commented:
Ok I've added the gene with a link to the HMM profile, but I left the "scope" column, as this is pretty important (if unclear). I've added a tooltip to the table to send users to the AMRFinderPlus documentation for more information on the fields.
